Add From Blender To OpenCV (#2)
expertisesolutions/expertisesolutions.github.io: _posts/2021-06-22-From-Blender-To-OpenCV.markdown

|  Y | left to right  | down to up    | up to down    | up to down    |
|  Z | down to up     | to the view   | to the view   | from the view |

# The transformation from Blender to OpenCV

To transform from the Blender View orientation to the OpenCV View orientation, 
we need to keep X while inverting Y and Z signals. But there is a trick here: 
if we want to describe an object in front of the camera, we will need to invert 
X's signal too, so we will use (X, -Y, -Z) if the object is the camera, 
but for another type, its (-X, -Y, -Z).

# Using the transformation
First, we can't use [Euler angles](https://en.wikipedia.org/wiki/Euler_angles) because of the same mathematical 
principle that causes "Gimbal Lock" ([read here](https://en.wikipedia.org/wiki/Gimbal_lock#Loss_of_a_degree_of_freedom_with_Euler_angles)).
So we will use the _rotation-translation matrix_.

Among other representations like quaternions, we choose to use the 
rotation-translation matrix to represent the pose of the object/camera. It is 
defined by a *rotation matrix* `R` and a *translation vector* `T`, both 
concatenated in a form of a `[R|T]` matrix. 

> For cameras, if the `[R|T]` matrix is further concatenated with the intrinsic 

Author: ```suggestion
For cameras, if the `[R|T]` matrix is further multiplied by the intrinsic 
```
